src/api/Buffer.test.ts

```diff
@@ -235,83 +264,87 @@ describe('Buffer API', () => {
+      withBuffer([], async buffer => {
+        buffer.replace('test', 0);
+        expect(await buffer.lines).toEqual(['test']);
+      })
+    );
+
+    it(
+      'can insert lines at beginning of buffer',
+      withBuffer(['test'], async buffer => {
+        buffer.insert(['test', 'foo'], 0);
+        expect(await buffer.lines).toEqual(['test', 'foo', 'test']);
+      })
+    );
+
+    it(
+      'can replace nvim.buffer starting at line 1',
+      withBuffer(['test', 'foo'], async buffer => {
+        await buffer.replace(['bar', 'bar', 'bar'], 1);
+        expect(await buffer.lines).toEqual(['test', 'bar', 'bar', 'bar']);
+      })
+    );
+
+    it(
+      'inserts line at index 2',
+      withBuffer(['test', 'bar', 'bar', 'bar'], async buffer => {
+        await buffer.insert(['foo'], 2);
+        expect(await buffer.lines).toEqual([
+          'test',
+          'bar',
+          'foo',
+          'bar',
+          'bar',
+        ]);
+      })
+    );
 
-    it('inserts line at index 2', withBuffer( 
-        [ 'test', 'bar', 'bar', 'bar' ],
-        async buffer => {
-            await buffer.insert(['foo'], 2);
-            expect(await buffer.lines).toEqual([
-                'test',
-                'bar',
-                'foo',
-                'bar',
-                'bar',
-            ]);
-            }
-    ));
-
-    it('removes last 2 lines', withBuffer( 
-        [ 'test', 'bar', 'foo', 'a', 'b' ],
-        async buffer => {
-            await buffer.remove(-3, -1);
-            expect(await buffer.lines).toEqual([ 'test', 'bar', 'foo' ]);
-        }
-    ));
-
-    it('append lines to end of buffer', withBuffer(
-    [ 'test', 'bar', 'foo' ], async buffer => {
-      await buffer.append(['test', 'test']);
-      expect(await buffer.lines).toEqual([
-        'test',
-        'bar',
-        'foo',
-        'test',
-        'test',
+    it(
+      'removes last 2 lines',
+      withBuffer(['test', 'bar', 'foo', 'a', 'b'], async buffer => {
+        await buffer.remove(-3, -1);
+        expect(await buffer.lines).toEqual(['test', 'bar', 'foo']);
+      })
+    );
+
+    it(
+      'append lines to end of buffer',
+      withBuffer(['test', 'bar', 'foo'], async buffer => {
+        await buffer.append(['test', 'test']);
+        expect(await buffer.lines).toEqual([
+          'test',
+          'bar',
+          'foo',
+          'test',
+          'test',
         ]);
-    }));
-
-    it('can clear the buffer', withBuffer( ['foo'], async buffer => {
-      await buffer.remove(0, -1);
-      // One empty line
-      expect(await buffer.length).toEqual(1);
-      expect(await buffer.lines).toEqual(['']);
-    }));
+      })
+    );
+
+    it(
+      'can clear the buffer',
+      withBuffer(['foo'], async buffer => {
+        await buffer.remove(0, -1);
+        // One empty line
+        expect(await buffer.length).toEqual(1);
+        expect(await buffer.lines).toEqual(['']);
+      })
+    );
+  });
```

Comment:
Added missing closing parenthesis and bracket to enclose `describe('Buffer API', ...` from https://github.com/neovim/node-client/pull/98/